cross/ifuse/Makefile

```diff
@@ -0,0 +1,24 @@
+PKG_NAME = ifuse
+PKG_VERS = 1.1.4
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/bxxxjxxg/$(PKG_NAME)/archive
```

Comment:
Version 1.1.4 does not officially exists. I propose you use commit hash as version and download from https://github.com/libimobiledevice/ifuse/archive/master.zip

Reply:
Thank you. It's useful. Now I can compile the upstream code up to a  specific commit.